### PR TITLE
dreambooth plus lora

### PR DESCRIPTION
**Pull request by wtomin**
Train DreamBooth with LoRA:
- [x] update `train_dreambooth.py` to support `lora_ft_unet` and `lora_ft_text_encoder`;
- [x] update `dreambooth_finetune.md`, increase db+lora training / inference commands, a figure illustrating db+lora generated images.
- [x] A new shell script: `run_train_dreambooth_w_lora_sd_v2.sh`.

### CODE REVIEW

File: examples/stable_diffusion_v2/dreambooth_finetune.md [hunk 1]
```diff
@@ -189,6 +189,38 @@ python train_dreambooth.py \
     --with_prior_preservation=False \
 ```
 
+#### 2.2.5 Training Command for DreamBooth with LoRA
+
+LoRA is a parameter-efficient finetuning method. Here, we combine DreamBooth with LoRA by injecting the LoRA parameters into the text encoder and the UNet of Text-to-Image model, and training with the prior preservation loss.
```
Comment (DongyaoZhu) [inline examples/stable_diffusion_v2/dreambooth_finetune.md:194]: during training? grammarly

**Review by DongyaoZhu — APPROVED** [review]
I only checked whether new changes affect old functionalities.

File: examples/stable_diffusion_v2/dreambooth_finetune.md (hunk 1 above)
Comment (wtomin) [reply examples/stable_diffusion_v2/dreambooth_finetune.md:194]: by injecting xxx, and training with xxx. It seems alright to me.

**Review by SamitHuang — APPROVED** [review]
lgtm

File: examples/stable_diffusion_v2/dreambooth_finetune.md (hunk 1 above)
Comment (DongyaoZhu) [reply examples/stable_diffusion_v2/dreambooth_finetune.md:194]: gucci just checked with gpt :D

**Pull request merged**